### PR TITLE
fix: fix k8s terminal set size

### PR DESCRIPTION
**Pull request by laojun**
## What this PR does / why we need it:
fix: fix k8s terminal set size

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required) 
release/1.3

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/common/components/tiled-filter.tsx [hunk 1]
```diff
@@ -41,10 +42,20 @@ export interface IProps {
...
-  const { fields, delay = 1000, value: propsValue, onChange, expand: propsExpand = true, labelWidth } = props;
+  const { fields, delay = 1000, value: propsValue, onChange, expand: propsExpand = true } = props;
   const [expand, setExpand] = React.useState(propsExpand);
   const [value, setValue] = React.useState(propsValue || {});
+  const [labelWidth, setLabelWidth] = React.useState<string | number>('auto');
+
+  useMount(() => {
+    const labelEles = document.querySelectorAll('.tiled-fields-item-label');
+    const maxWidth: number = max(map(labelEles, (ele: HTMLSpanElement) => ele.offsetWidth));
+    setLabelWidth(maxWidth > MAX_LABEL_WIDTH ? MAX_LABEL_WIDTH : maxWidth);
```
Comment (daskyrk) [inline shell/app/common/components/tiled-filter.tsx:56]: `Math.min(a, b)` is better

File: shell/app/configForm/editor/form-preview.tsx [hunk 2]
```diff
@@ -410,6 +410,9 @@ const PreviewForm = (props: any) => {
   const formConfig = React.useMemo(() => fields, [fields]);
 
   const onFinish = (data: any) => {
+    // used log
+    // eslint-disable-next-line no-console
+    console.log('------', data);
```
Comment (daskyrk) [inline shell/app/configForm/editor/form-preview.tsx:415]: Is this required?

Comment (laojun) [reply shell/app/configForm/editor/form-preview.tsx:415]: yes, it's a form data for form-preview

**Review by McDaddy — APPROVED** [review]

**Pull request merged**